ONBOARDING.md

```diff
@@ -0,0 +1,84 @@
+- [Introduction](#introduction)
+  - [Who this is for](#who-this-is-for)
+  - [What this guide will help you do](#what-this-guide-will-help-you-do)
+- [Learn key concepts](#learn-key-concepts)
+  - [About the OpenSearch project](#about-the-opensearch-project)
+  - [How we work](#how-we-work)
+  - [Contributor expectations](#contributor-expectations)
+  - [Resources and links](#resources-and-links)
+- [Set up accounts and access](#set-up-accounts-and-access)
+  - [GitHub account](#github-account)
+  - [OpenSearch forum account](#opensearch-forum-account)
+
+# Getting started as an OpenSearch contributor
```

Comment:
Nit: can we Capitalize Titles Like in Other Markdown Files?

Reply:
lol, sure - I'll also double check the style guide for other things I may have missed. Sentence case titles are a strong personal preference, but consistency is better 😉

Reply:
At least the CONTRIBUTING uses cap-titles.

Reply:
I'll defer to maintainers of this repo over whether we choose title casing to match other markdown files here, or whether it's better for new documents to follow the project style guide: https://github.com/opensearch-project/documentation-website/blob/main/STYLE_GUIDE.md#punctuation-and-capitalization

> Use sentence case for titles, headings, and table headers. Titles of standalone documents may use title case.

Reply:
I'd be happy to review a PR for a markdown linter if we'd like to codify these styles